solr/contrib/analytics/src/java/org/apache/solr/analytics/facet/AbstractSolrQueryFacet.java

```diff
@@ -50,11 +49,11 @@ protected AbstractSolrQueryFacet(String name) {
    *
    * Each of these executors will be executed after the streaming phase in the {@link AnalyticsDriver}.
    *
-   * @param filter the overall filter representing the documents being used for the analytics request
```

Comment:
I think the name "filter" is reasonable as it conveys it's purpose is only for filtering and not also for scoring.  If possible, we could use DocSet?

Reply:
Agreed!

Reply:
I made the change to `DocSet` in `AnalyticsDriver` but not in other areas because the change from `Query` to `DocSet` affects a lot of other APIs further downstream. I left a TODO comment in `AnalyticsDriver` indicating this. Maybe we can punt the changes to a future issue?

Reply:
Looks good to me as you have it; thanks.